modules/ocf_irc/files/anope/chanserv.conf

```diff
@@ -0,0 +1,172 @@
+  channels = "#services"
+}
+
+module {
+  name = "chanserv"
+  client = "ChanServ"
+
+  defaults = "keeptopic peace cs_secure securefounder signkick"
+  expire = 120d
+
+  reasonmax = 200
+  signkickformat = "%m (%n)"
+}
+
+# Privileges
+privilege { name = "ACCESS_CHANGE"; rank = 0;   level = 10;        flag = "f"; xop = "SOP"; }
+privilege { name = "ACCESS_LIST";   rank = 10;  level = 3;         flag = "f"; xop = "VOP"; }
+privilege { name = "AKICK";         rank = 250; level = 10;        flag = "K"; xop = "SOP"; }
+privilege { name = "AUTOHALFOP";    rank = 100; level = 4;         flag = "H"; xop = "HOP"; }
+privilege { name = "AUTOOP";        rank = 210; level = 5;         flag = "O"; xop = "AOP"; }
+privilege { name = "AUTOOWNER";     rank = 330; level = 9999;      flag = "Q"; xop = "QOP"; }
+privilege { name = "AUTOPROTECT";   rank = 240; level = 10;        flag = "A"; xop = "SOP"; }
+privilege { name = "AUTOVOICE";     rank = 50;  level = 3;         flag = "V"; xop = "VOP"; }
+privilege { name = "BAN";           rank = 150; level = 4;         flag = "b"; xop = "HOP"; }
+privilege { name = "FOUNDER";       rank = 360; level = 10000;     flag = "F"; xop = "QOP"; }
+privilege { name = "GETKEY";        rank = 180; level = 5;         flag = "G"; xop = "AOP"; }
+privilege { name = "HALFOP";        rank = 120; level = 5;         flag = "h"; xop = "AOP"; }
+privilege { name = "HALFOPME";      rank = 110; level = 4;         flag = "h"; xop = "HOP"; }
+privilege { name = "INFO";          rank = 80;  level = 9999;      flag = "I"; xop = "QOP"; }
+privilege { name = "INVITE";        rank = 190; level = 5;         flag = "i"; xop = "AOP"; }
+privilege { name = "KICK";          rank = 130; level = 4;         flag = "k"; xop = "HOP"; }
+privilege { name = "MODE";          rank = 170; level = 9999;      flag = "s"; xop = "QOP"; }
+privilege { name = "OP";            rank = 230; level = 5;         flag = "o"; xop = "SOP"; }
+privilege { name = "OPME";          rank = 220; level = 5;         flag = "o"; xop = "AOP"; }
+privilege { name = "OWNER";         rank = 350; level = "founder"; flag = "q"; xop = "QOP"; }
+privilege { name = "OWNERME";       rank = 340; level = 9999;      flag = "q"; xop = "QOP"; }
+privilege { name = "PROTECT";       rank = 310; level = 9999;      flag = "a"; xop = "QOP"; }
+privilege { name = "PROTECTME";     rank = 300; level = 10;        flag = "a"; xop = "AOP"; }
+privilege { name = "SET";           rank = 320; level = 9999;      flag = "s"; xop = "QOP"; }
+privilege { name = "SIGNKICK";      rank = 140; level = 9999;      flag = "K"; xop = "QOP"; }
+privilege { name = "TOPIC";         rank = 160; level = 5;         flag = "t"; xop = "AOP"; }
+privilege { name = "UNBAN";         rank = 200; level = 4;         flag = "u"; xop = "HOP"; }
+privilege { name = "VOICE";         rank = 70;  level = 4;         flag = "v"; xop = "HOP"; }
+privilege { name = "VOICEME";       rank = 60;  level = 3;         flag = "v"; xop = "VOP"; }
```

Comment:
This file is a lot shorter now, mostly just because I compacted these lines though. Still, I think it makes it easier to read and edit easily.